modules/payloads/singles/ruby/pingback_bind_tcp.rb

```diff
@@ -0,0 +1,55 @@
+require 'msf/core/handler/bind_tcp'
+require 'msf/core/payload/ruby'
+
+require 'msf/base/sessions/pingback'
+require 'msf/core/payload/pingback'
+
+module MetasploitModule
+
+  CachedSize = 147
+
+  include Msf::Payload::Single
+  include Msf::Payload::Ruby
+  include Msf::Payload::Pingback::Options
+
+  def initialize(info = {})
+    super(merge_info(info,
+      'Name' => 'Ruby Pingback, Bind TCP',
+      'Description' => 'Listens for a connection from the attacker, sends a UUID, then terminates',
+      'Author' => 'asoto-r7',
+      'License' => MSF_LICENSE,
+      'Platform' => 'ruby',
+      'Arch' => ARCH_RUBY,
+      'Handler' => Msf::Handler::BindTcp,
+      'Session' => Msf::Sessions::Pingback,
+      'PayloadType' => 'ruby',
+      'Payload' =>
+        {
+          'Offsets' => { },
+          'Payload' => ''
+        }
+      ))
+  end
+
+  def generate
+    #return prepends(ruby_string)
+    return ruby_string
+  end
+
+  def ruby_string
+    self.pingback_uuid ||= self.generate_pingback_uuid
+
+    return "require 'socket';"+
```

Comment:
Instead of using `+` we can format this like:
```ruby
return "require 'socket'" \
  "s=TCPServer.new(#{datastore['LPORT'].to_i});" \
  ...
```
Which is much more efficient since it creates only one string and doesn't do a bunch of copying. https://github.com/rubocop-hq/ruby-style-guide#long-strings